src/resources/packages/classy/store/selectors.ts

```diff
@@ -103,7 +103,7 @@ export function getEditedPostVenueIds(): number[] {
  * @return {boolean} Whether tickets are supported.
  */
 export function areTicketsSupported( state: StoreState ): boolean {
-	return state?.areTicketsSupported || false;
+	return state?.areTicketsSupported || true;
```

Comment:
I guess this is set to `true` for debugging purposes, I think it should be `false` by default, right?

Reply:
Correct, the default will be restored back to `false` once ET is able to impact the setting.